plugin/pkg/auth/authenticator/request/x509/x509.go

```diff
@@ -104,7 +105,13 @@ var DNSNameUserConversion = UserConversionFunc(func(chain []*x509.Certificate) (
 
 // EmailAddressUserConversion builds user info from a certificate chain using the first EmailAddress on the certificate
 var EmailAddressUserConversion = UserConversionFunc(func(chain []*x509.Certificate) (user.Info, bool, error) {
```

Comment:
Where is this function called?  In `pkg/apiserver/authenticator/authn.go` I see `newAuthenticatorFromClientCAFile` does `return x509.New(opts, x509.CommonNameUserConversion), nil`.  But I don't see where `EmailAddressUserConversion` is ever referenced.